docs/appendix/troubleshooting.md

```diff
@@ -225,21 +225,21 @@ This file contains basic diagnostic and configuration information about your cam
 Under **Config > Components**, find your camera, then fill in your camera's specific configuration either using the **Show more** button to show the relevant configuration options, or the **Go to advanced** link in the component panel's upper-right to enter these attributes manually.
 Provide at least the width and height values to start.
 
-## Known Application and Plugin Conflicts
+## Known application and plugin conflicts
 
-### macOS Applications
+### macOS applications
 
 None at this time.
 
-### Windows Applications
+### Windows applications
 
 None at this time.
 
-### Linux Applications
+### Linux applications
 
 None at this time.
 
-### Browser Plugins
+### Browser plugins
 
-**Chrome Plugin: Allow right click** - This Chrome plugin interferes with the [Viam app](https://app.viam.com)'s ability to configure a service.
+**Chrome plugin: allow right click** - This Chrome plugin interferes with the [Viam app](https://app.viam.com)'s ability to configure a service.
```

Comment:
```suggestion
**Chrome plugin: Allow Right-Click** - This Chrome plugin interferes with the [Viam app](https://app.viam.com)'s ability to configure a service.
```
It appears that this (with a hyphen) is the actual name of it, so it's a proper noun